tests/conftest.py

```diff
@@ -236,7 +237,8 @@ def _builder(
 @pytest.fixture
 def lesson_page_builder():
     def _builder(
-      id, title, html_content, prevpageid=1, nextpageid=1, answers=[]
+      id, title, html_content, qtype, prevpageid=1, nextpageid=1,
```

Comment:
Instead of the `.get('qtype', 20)` used when invoking `lesson_page_builder` on L289, what do you think about making it optional by setting a default value here?

Reply:
Per live discussion, keeping this approach is probably the simpler alternative. Resolving this comment accordingly.